assets/terraform-modules/packet/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -102,7 +103,12 @@ resource "null_resource" "bootkube-start" {
 
   provisioner "remote-exec" {
     inline = [
-      "sudo mv $HOME/assets /opt/bootkube",
+      "sudo mv $HOME/assets /opt/bootkube/",
+      # This is needed, as bootkube-start script will move all files matching
+      # /opt/bootkube/asssets/manifests-*/*.yaml into /opt/bootkube/assets/manifests.
+      #
+      # -p is not needed here, as we create assets directory in the command above.
```

Comment:
I think we can remove the `-p` explanation, it was just me missing the previous commands
```suggestion
```